app/_includes/components/new_in.html

```diff
@@ -0,0 +1,2 @@
+{%- capture text -%}New in: {{version}}{%- endcapture -%}
```

Comment:
```suggestion
{%- capture text -%}v{{version}}+{%- endcapture -%}
```

"New" is temporary - if we're 8 versions later it's no longer new.